src/java/utils/JsonHelper.java

```diff
@@ -0,0 +1,17 @@
+package utils;
+
+import net.sf.json.JSONObject;
+
+/**
+ *
+ * @author markskroba
+ */
+public class JsonHelper {
+    public static JSONObject buildNoneLanguageMap(String label)
```

Comment:
As this integrates with the `buildPage()` the type of @return may change, but this is meant to be a straightforward start and it is done right.